src/core/exec-invoke.c

```diff
@@ -369,10 +369,22 @@ static int setup_input(
         case EXEC_INPUT_TTY_FORCE:
         case EXEC_INPUT_TTY_FAIL: {
                 _cleanup_close_ int tty_fd = -EBADF;
+                _cleanup_free_ char *resolved = NULL;
```

Comment:
Hmm, the resolved tty here isn't really reflected on pam, which still preferably uses `context->tty_path`?